test/services/sync/strategies/TSP/TSP.integration.test.js

```diff
@@ -0,0 +1,42 @@
+const { expect } = require('chai');
+
+const app = require('../../../../../src/app');
+
+const testObjects = require('../../../helpers/testObjects')(app);
+const { generateRequestParamsFromUser } = require('../../../helpers/services/login')(app);
+const { create: createUser } = require('../../../helpers/services/users')(app);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['createUser' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5083752)